node_cli/operations/base.py

```diff
@@ -177,7 +179,7 @@ def migrate_mirage_boot(env_filepath: str, env: Dict) -> bool:
         )
         docker_cleanup()
 
-    update_meta(
+    meta_manager.update_meta(
```

Comment:
why we're using `CliMetaManager` and not `MirageCliMetaManager` for migrate mirage boot and init mirage boot?